products/memgraph/upcoming/tutorials/football-transfers.md

```diff
@@ -0,0 +1,274 @@
+  --storage-snapshot-on-exit=false --storage-properties-on-edges=true
+```
+
+The user should note that any modifications of the database state will persist
+only during this run of Memgraph.
+
+### Example Queries using OpenCypher
+
+In the queries below, we are using [OpenCypher](https://www.opencypher.org) 
+to query Memgraph via the console.
+
+
+1) Let's say you want to find 20 most expensive transfers.
+As mentioned before fee is represented in millions of euros.
+
+```opencypher
+MATCH (t:Transfer)<-[:TRANSFERRED_IN]-(p:Player)
+WHERE t.fee is NOT NULL
+RETURN  t.fee as transfer_fee ,p.name AS player_name
+ORDER BY t.fee DESC LIMIT 20
+```
+
+2) What about finding the most expensive transfer per season? 
+
+```opencypher
+MATCH path = (s:Season)<-[:HAPPENED_IN]-(t:Transfer)<-[:TRANSFERRED_IN]-(p:Player)
+WHERE t.fee is NOT NULL
+WITH  s.name as season, MAX(t.fee) as maxFee
+RETURN  maxFee,season
+ORDER BY maxFee DESC
+```
+
+
+3)How about finding out for which teams has your favorite player played for?
+Instead of Sime Vrsaljko, you can instead put the name of your favorite player.
+
+```opencypher
+MATCH (p:Player)-[:TRANSFERRED_IN]->(t:Transfer)-[]-(m:Team)
+WHERE p.name="Sime Vrsaljko"
+WITH DISTINCT m
+RETURN m.name AS team_name
+```
+
+4)Find players that were transferred to and played for FC Barcelona and
+count them by the position they have in the game.
+
+```opencypher
+MATCH (m:Team)<-[:TRANSFERRED_TO]-(t:Transfer)<-[:TRANSFERRED_IN]-(p:Player)
+WHERE m.name="FC Barcelona"
+WITH * ORDER BY p.name
+WITH DISTINCT p
+RETURN p.position as position, COUNT(p) AS position_count, collect(p.name) as playerNames
+ORDER BY position_count DESC
+```
+
+5) Football has seen a lot of rivalries develop between clubs during its rich and long history,
+some of which can be traced back more than a century. Now one of the most famous ones is between
+FC Barcelona and Real Madrid. Let's find out were there any transfers between FC Barcelona and Real Madrid.
+
+```opencypher
+MATCH path_teams = (m:Team )-[:TRANSFERRED_FROM]-(t:Transfer)-[:TRANSFERRED_TO]-(n:Team)
+WHERE (m.name="FC Barcelona" AND n.name="Real Madrid") OR (m.name="Real Madrid" AND n.name="FC Barcelona")
+MATCH path_player = (t)<-[:TRANSFERRED_IN]-(p:Player)RETURN path_teams,path_player
+```
+
+
+6) FC Barcelona is one of the biggest clubs in the world. Players often want to stay there as long as possible.
+But what about those players who didn't fit in well? Where do they go?
+
+```opencypher
+MATCH (m:Team )-[:TRANSFERRED_FROM]->(t:Transfer)<-[:TRANSFERRED_IN]-(p:Player), (t)-[:TRANSFERRED_TO]->(n:Team)
+WHERE m.name="FC Barcelona"
+RETURN n.name as team_name, collect(p.name) as names, COUNT(p) AS number_of_players
+ORDER BY number_of_players DESC
+```
+
+
+7) What are the teams that most players went to in season 2003/2004? The results may surprise you. 
+
+```opencypher 
+MATCH (s:Season {name:"2003/2004"})<-[:HAPPENED_IN]-(t:Transfer)<-[:TRANSFERRED_IN]-(player:Player)
```

Comment:
As in other queries, move this name in WHERE to be consistent.

Reply:
:+1: